netlify.toml

```diff
@@ -1,4 +1,4 @@
 [build]
   # If there was no difference between the last two commits 
   # in one of the sub directories do not build the page again
-  ignore = "git diff --quiet HEAD^ HEAD data/ images/ pages/ scripts/"
+  ignore = "git diff --quiet HEAD^ HEAD -- data/ images/ pages/ scripts/"
```

Comment:
@yamalight FYI
This command seems to work pretty well. The commit only changes the netlify file and the [build is aborted](https://app.netlify.com/sites/dice-website-preview/deploys/5e174c6a7d730e0008972133). Only if there are changes within the 4 directories listed above the build takes place.